licenses/management/commands/publish.py

```diff
@@ -0,0 +1,148 @@
+import os
+import subprocess
+from argparse import ArgumentParser
+
+from django.core.management import BaseCommand
+
+from cc_licenses.settings.base import TRANSLATION_REPOSITORY_DIRECTORY
+
+# Go to cc-licenses-data and do something
+GO_TO_TRANSLATIONS_REPO = f"cd {TRANSLATION_REPOSITORY_DIRECTORY} && "
+PULL_BRANCHES_DOWN_FROM_DEVELOP = "git checkout develop && git pull"
+PULL_DOWN_TRANSLATION_BRANCHES = (
+    GO_TO_TRANSLATIONS_REPO + PULL_BRANCHES_DOWN_FROM_DEVELOP
+)
+
+
+def run_django_distill():
+    """Outputs the build directory
+    """
```

Comment:
Does this docstring mean it prints out the path to the build directory when it runs?

Reply:
updated:
```python
 """Outputs static files into the specified directory determined by settings.base.DISTILL_DIR
 """
```